paddle/fluid/operators/reader/create_recordio_file_reader_op.cc

```diff
@@ -65,20 +65,22 @@ class CreateRecordIOReaderOp : public framework::OperatorBase {
                       static_cast<int>(shape_concat.size()),
                       "The accumulate of all ranks should be equal to the "
                       "shape concat's length.");
-    std::string filename = Attr<std::string>("filename");
+    auto filenames = Attr<std::vector<std::string>>("filenames");
```

Comment:
`fluid.layers.open_files` should do the job, please see `test_multi_file_reader.py`

Reply:
Seems `open_files_op` can only deal with the normal files.

Reply:
`open_files_op` can deal with RecordIO file. What's your 'unnormal' file?